src/grid.rs

```diff
@@ -0,0 +1,269 @@
+use std::thread::sleep;
+use std::time::Duration;
+
+pub struct Grid {
+    matrix: Vec<Vec<u8>>,
```

Comment:
This works but maybe it would be a better alternative to use other type of value for representing cell status like a boolean or an enum with 2 possible values. Using u8 for representing only 2 states may be a little bit overkill, but that's just my POV

Reply:
Yes, I preferred an u8 because initially because to print the values on the grid it was easier to print an 1 or 0 than a true or false, but now its better a boolean value.